drivers/SmartThings/sonos/src/api/cmd_handlers.lua

```diff
@@ -151,7 +153,8 @@ function CapCommandHandlers.handle_play_preset(driver, device, cmd)
     { namespace = "favorites", command = "loadFavorite" },
     {
       favoriteId = cmd.args.presetId,
-      playOnCompletion = true
+      playOnCompletion = true,
+      action = (device.preferences[QUEUE_ACTION_PREF] or "APPEND")
```

Comment:
Is there capitalization that needs to be accounted for when using the ST preference value? Looks like the default `"APPEND"` is gonna be different that the actual table value.

Reply:
The API itself is case-insensitive on stringly-defined types.